lib/kinesis/consumer.rb

```diff
@@ -3,6 +3,7 @@
 require 'concurrent/hash'
 require 'kinesis/shard_reader'
 require 'kinesis/state'
+require 'logger'
```

Comment:
Maybe silly question, how does this work?

Reply:
Yes, standard library. This is actually at its most basic, do you have a suggestion for a better one? I do expect people to pass in Rails.logger anyway haha. Not sure if I'm going about this the right way.